HLTrigger/Configuration/python/customizeHLTfor2016trackingTemplate.py

```diff
@@ -0,0 +1,686 @@
+             ),
+             minNVtxTrk = cms.int32(3),
+             maxDz = cms.vdouble(
+               0.5,
+               0.2,
+         #      0.2
+               3.4028234663852886e+38
+             ),
+             maxDzWrtBS = cms.vdouble(
+               3.4028234663852886e+38,
+               24,
+               15
+             ),
+             maxDr = cms.vdouble(
+               0.5,
+               0.03,
+         #      0.03
+               3.4028234663852886e+38
+             ),
+             dz_par = cms.PSet(
+                     dz_exp = cms.vint32(
+                         4,
+                         4,
+                         4,
+                     ),
+                     dz_par1 = cms.vdouble(
+                         0.4,
+                         0.4,
+                         0.4
+         #                3.4028234663852886e+38
+                     ),
+                     dz_par2 = cms.vdouble(
+                         0.35,
+                         0.35,
+                         0.35
+         #                3.4028234663852886e+38
+                     ),
+             ),
+             dr_par = cms.PSet(
+                     dr_exp = cms.vint32(
+                         4,
+                         4,
+                         4,
+                     ),
+                     dr_par1 = cms.vdouble(
+                         0.4,
+                         0.4,
+                         0.4
+         #                3.4028234663852886e+38
+                     ),
+                     dr_par2 = cms.vdouble(
+                         0.3,
+                         0.3,
+                         0.3
+         #                3.4028234663852886e+38
+                     ),
+               d0err = cms.vdouble(
+                 0.003,
+                 0.003,
+                 0.003
+               ),
+               d0err_par = cms.vdouble(
+                 0.001,
+                 0.001,
+                 0.001
+               )
+             )
+           )
+         )
+        )        
+        setattr(process,'hltIter0PFlowTrackSelectionHighPurity', cms.EDProducer( "TrackCollectionFilterCloner",
+          originalSource   = cms.InputTag('hltIter0PFlowCtfWithMaterialTracks'),
+          originalMVAVals  = cms.InputTag('hltIter0PFlowTrackCutClassifier','MVAValues'),
+          originalQualVals = cms.InputTag('hltIter0PFlowTrackCutClassifier','QualityMasks'),
+          minQuality       = cms.string('highPurity') ,
+          cloner = cms.untracked.PSet(
+            copyExtras       = cms.untracked.bool(False),
+            copyTrajectories = cms.untracked.bool(False)
+          )
+         )
+        )
+        iter0HP  = getattr(process,'hltIter0PFlowTrackSelectionHighPurity')
+
+        iter0seq = getattr(process,'HLTIterativeTrackingIteration0')
+        iter0seq.insert( iter0seq.index( iter0HP ), getattr(process,'hltIter0PFlowTrackCutClassifier') )
+        print 'iter0seq: ', iter0seq
```

Comment:
Please remove this and all other "print" statements in the py files.
The printouts would end up in the final py file and lead to error as they are not valid python.
In any case the py file should be "silent"